src/table/body/table-row.component.ts

```diff
@@ -59,7 +59,9 @@ import { Observable } from "rxjs";
 					[ngStyle]="model.getHeader(j).style"
 					[skeleton]="skeleton"
 					[attr.colspan]="item.colSpan"
-					[attr.rowspan]="item.rowSpan">
+					[attr.rowspan]="item.rowSpan"
+					(click)="onRowClick()"
```

Comment:
So adding these once on `tr` instead of every `td` doesn't work?

Reply:
Adding them once on `tr` would also get triggered if a selection/expansion column was clicked